conventions/Documentation.md

```diff
@@ -34,66 +34,76 @@ You can provide code snippets demonstrating a specific behavior or pattern for a
+import accordionItemReadme from "../accordion-item/readme.md";
 
-#### General
+export default {
+  title: "Components/Accordion",
+  parameters: {
+    notes: {
+      accordion: accordionReadme,
+      accordionItem: accordionItemReadme
+    },
+    ...
+};
+```
+
+#### General guidelines
 
 - Story names should be camelCased
 - All stories need to import the `storyFilters` util and include it in the default export so stories can be properly filtered
-  ```ts
-  export default {
-    title: "Components/My Component",
-    parameters: {
-      notes: readme
-    },
-    ...storyFilters()
-  };
-  ```
 - Update the [main custom theme](https://github.com/Esri/calcite-components/blob/master/src/05-custom-theme.stories.mdx) story instead of adding a specific story showing how to use a custom CSS prop
 - Should only have HTML for the component or use case itself (e.g., no need to wrap in calcite-label)
 - Update the `simple` story with a corresponding knob instead of adding a story specific to a new prop
-- Skip adding or updating a story if it is covered by an existing one
+- Don't add or update a story if it is covered by an existing one
 
 #### Interactive showcase stories
 
 - Should have knobs configured to allow users to toggle different options available to the component
-- The simplest and base configuration should be named `simple` for consistency
+- The simplest, base configuration should be named `simple` for consistency
 - This story should have as many knobs possible for developers to interact with
 - Should have one story per unique, supported use case
-- Stories that can be covered by adding a new prop/knob should update an existing playground story
+- Stories that can be covered by adding a new prop/knob should instead update an existing playground story (with the exception of `_TestOnly` stories, explained below)
 
 #### Screenshot test-only stories
 
-- Should have the minimal HTML to reproduce the test scenario
-- Should not have knobs as screenshot tests will not interact with them
-- Stories that are only meant for testing should use the `_TestOnly` suffix, which ensures it is only used for the screenshot test build
-- Stories that are not meant for testing (e.g., require interaction not supported by the screenshot test environment) should use the `_NoTest` suffix, which ensures it is only used for the internal playground build
-- The `simple` story and any other significant configuration should have a matching `darkThemeRtl_TestOnly` story
+- Should have the minimal HTML needed to reproduce the test scenario
+- Should not have knobs since screenshot tests cannot interact with them
+- Stories that are only meant for testing should use the `_TestOnly` suffix, which ensures it is only used for the screenshot test build. For example, if a snapshot with specific properties is needed for testing, it wouldn't need to be the playground where knobs can be used to display the state in the `simple` story.
```

Comment:
> it wouldn't need to be the playground

Seems like this is missing something between `to be` and `the playground`.

WDYT about adding `displayed in` or `included in`?